tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -873,10 +873,7 @@ public function testListTableWithBlob()
      */
     protected function createTestTable($name = 'test_table', $data = array())
     {
-        $options = array();
-        if (isset($data['options'])) {
-            $options = $data['options'];
-        }
+        $options = $data['options'] ?? array();
```

Comment:
Do you want to use the short array notation for consistency with the previous code style changes?

Reply:
I'll use. I did not change it to avoid multiple changes in one PR :sweat_smile: